projects/canopy/src/lib/carousel/carousel.component.html

```diff
@@ -0,0 +1,59 @@
+<section class="lg-carousel" [attr.aria-labelledby]="description">
+  <lg-heading [level]="headingLevel" class="lg-carousel__heading lg-visually-hidden">{{
+    description
+  }}</lg-heading>
+  <div
+    class="lg-carousel__active-content lg-visually-hidden"
+    aria-atomic="true"
+    aria-live="polite"
+    [innerHTML]="selectedItemContent"
+  ></div>
+  <div
+    class="lg-carousel__wrapper"
+    [style.width.%]="100 * carouselItemCount"
+    [style.left.%]="-100 * selectedItemIndex"
+    [style.transition]="'left ease ' + slideDuration / 1000 + 's'"
+  >
+    <ng-content></ng-content>
+  </div>
+
+  <div class="lg-carousel__navigation" role="tablist">
+    <button
+      role="tab"
+      class="lg-carousel__button"
+      (click)="previousCarouselItem()"
+      [disabled]="!loopMode && selectedItemIndex === 0"
+    >
+      <span class="lg-visually-hidden">Previous {{ description }} item</span>
+      <span class="lg-carousel__button__disk lg-carousel__centered">
+        <lg-icon name="chevron-left"></lg-icon>
+      </span>
+    </button>
+    <button
+      role="tab"
+      class="lg-carousel__button lg-carousel__centered"
```

Comment:
Wondering if `lg-carousel__centered` should be a modifier class? E.g. `lg-carousel__button--centered`

Reply:
Yes this should indeed be a modifier class